components/collector/src/source_collectors/trello/issues.py

```diff
@@ -19,30 +19,33 @@ async def _parse_entities(self, responses: SourceResponses) -> Entities:
         json = await responses[0].json()
         cards = json["cards"]
         lists = {lst["id"]: lst["name"] for lst in json["lists"]}
-        return Entities(self.__card_to_entity(card, lists) for card in cards if not self.__ignore_card(card, lists))
+        return Entities(self.__card_to_entity(card, lists) for card in cards)
 
-    def __ignore_card(self, card, lists) -> bool:
-        """Return whether the card should be ignored."""
+    def _include_entity(self, entity: Entity) -> bool:
+        """Return whether the card should be included."""
 
         def card_is_inactive() -> bool:
             """Return whether the card is inactive."""
-            date_last_activity = parse(card["dateLastActivity"])
+            date_last_activity = parse(entity["date_last_activity"])
             return days_ago(date_last_activity) > int(cast(int, self._parameter("inactive_days")))
 
         def card_is_overdue() -> bool:
             """Return whether the card is overdue."""
-            due_date = parse(card["due"]) if card.get("due") else datetime.max
+            due_date = parse(entity["due_date"]) if entity["due_date"] else datetime.max
             return due_date < datetime.now(tz=due_date.tzinfo)
 
         lists_to_ignore = self._parameter("lists_to_ignore")
-        if card["idList"] in lists_to_ignore or lists[card["idList"]] in lists_to_ignore:
-            return True
+        if entity["id_list"] in lists_to_ignore or entity["list"] in lists_to_ignore:
+            return False
+
         cards_to_count = self._parameter("cards_to_count")
         if "overdue" in cards_to_count and card_is_overdue():
-            return False
+            return True
         if "inactive" in cards_to_count and card_is_inactive():
-            return False
-        return bool(cards_to_count)
+            return True
+        # TODO - warrants comment or logic change: do we only include if we don't count overdue nor inactive?
+        #  Doesn't this mean that inactive and overdue cards are always counted when we are not supposed to count them?
+        return not bool(cards_to_count)
```

Comment:
If you change a ignore() method to include() and the ignore() method has two checks, you should not only invert the checks, but also reverse to order of the steps to get the exact same semantics I think? I should write a test program, but I'm too lazy ;-)

Reply:
I tried the following approach but it results in several failing tests (even if I keep the botton return as `not bool`):
```
if "overdue" not in cards_to_count and card_is_overdue():
  return False
if "inactive" not in cards_to_count and card_is_inactive():
  return False
return True
```

Reply:
Ah, I see the issue.

Reply:
Hmm but I don't think I understand the testcases either. The fixture returns 2 values, but when overdue or inactive is turned _on_ it actually returns less data?